Parkdale-Digital/shopify-webcomponents-test
# Update product card files and clean up assets

### Overview
This PR implements the product card component using Web Components and Liquid templating. The `<product-card>` Web Component displays product data (title, price, image, URL) using attributes passed from the Liquid template.

### Changes:
- Created `main.js` for the Web Component with Shadow DOM encapsulation.
- Styled the product card using SCSS in `main.scss`.
- Added a Liquid section file (`product-card.liquid`) to render the component and pass product data.
- Implemented an "Add to Cart" button with quantity selection.

### Testing:
- Verified functionality on the Shopify dev store with different products.
- Tested responsiveness across mobile, tablet, and desktop views.
- Checked cross-browser compatibility (Chrome, Firefox, Safari, Edge).
- Ensured accessibility with proper semantic HTML and keyboard navigation.

### Notes:
- The original repository did not have a `dev` branch, so this pull request is being submitted to the `main` branch.
- Used a placeholder image as a fallback if no image is provided.

### How to Test:
1. Add the "Product Card" section to a page using the Shopify theme customizer.
2. Select a product in the section settings.
3. Verify that the product card displays the correct data and styling.
4. Test the "Add to Cart" functionality with different quantities.
